### ISSUE TITLE
Listbuilder: view switcher menu can't be used with keyboard (2.1.1, 2.4.3)

### ISSUE THREAD

**Issue by Blackbaud-MattGregg** [blackbaud/skyux1#950]
### Expected behavior
Upon opening the view switcher menu, the keyboard can be used to move focus into the menu and select the menu items.

### Actual behavior
The keyboard can't be used to access the menu items. The menu is inserted at the end of the DOM.
